packages-exp/functions-exp/src/api.ts

```diff
@@ -55,6 +56,23 @@ export function getFunctions(
   return functionsInstance;
 }
 
+/**
+ * Modify this instance to communicate with the Cloudd Functions emulator.
+ *
+ * <p> Note: this must be called before this instance has been used to do any operations.
```

Comment:
Does the <p> tag do something on devsite?

Reply:
Nope, just JavaDoc habits.